docs/debugger/hot-reload.md

```diff
@@ -74,7 +74,7 @@ This experience is powered by native Edit and Continue. For supported edits, see
 When using Visual Studio 2022 and starting the app with the debugger, Hot Reload works with most app frameworks, including typical app types such as Console, Windows Forms (WinForms), WPF, UWP, WinUI 3 (see note), and most types of ASP.NET web projects (for code-behind edits) including ASP.NET MVC, Web API, and even older Web Forms projects. These are examples. Anywhere you have .NET and you’re using the Visual Studio managed debugger, you should get basic Hot Reload support. This fact means that even projects such as Azure Functions work great in this scenario.
 
 > [!NOTE]
-> WinUI 3 by default uses mixed mode debugging, which does not support Hot Reload. You can modify this in project settings by enabling the Managed Debugger, which enables Hot Reload to work properly.
+> WinUI 3 by default uses mixed mode debugging, which does not support Hot Reload. You can modify this in project settings by enabling the Managed Debugger, which enables Hot Reload to work properly. To enable this in your project modify launchSettings.json and add `"nativeDebugging": false` under commandName property
```

Comment:
```suggestion
> WinUI 3 by default uses mixed mode debugging, which does not support Hot Reload. You can modify this in project settings by enabling the Managed Debugger, which enables Hot Reload to work properly. To enable this in your project modify launchSettings.json and add `"nativeDebugging": false` after the `commandName` property
```

Reply:
I'd say "after" instead of under to try to avoid implying that it's nested. As far as the example, maybe that would be helpful. If commandName is always present in all WinUI scenarios, it's probably OK and people might not need it.